odoo-shippable/scripts/build-image.sh

```diff
@@ -241,8 +242,13 @@ endif
 EOF
 
 cat >> ~/.vimrc.before.local << EOF
+# Disable youcompleteme for default
+#let g:spf13_bundle_groups = ['general', 'writing', 'odoovim', 'wakatime',
+#                           \ 'programming', 'youcompleteme', 'php', 'ruby',
+#                           \ 'python', 'javascript', 'html',
+#                           \ 'misc']
 let g:spf13_bundle_groups = ['general', 'writing', 'odoovim', 'wakatime',
-                           \ 'programming', 'youcompleteme', 'php', 'ruby',
```

Comment:
@moylop260 
I try to compile the `youcompleteme` but the version of `g++` is not enought, the minimun version required is `g++4.9`

![image](https://user-images.githubusercontent.com/1387970/27394908-f70f3004-567b-11e7-9083-cd0e50918714.png)

In this PR only download the source code of `youcompleteme`